tools/SwiftSyntax/CMakeLists.txt

```diff
@@ -1,5 +1,29 @@
+set(swiftsyntax_compile_flags "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}")
+
+# FIXME: This is _awful_. These pieces could/should be sunk into
+#        `add_swift_library`
+if("${SWIFT_HOST_VARIANT_SDK}" STREQUAL "LINUX")
+  LIST(APPEND swiftsyntax_compile_flags
+    -I"${SWIFT_PATH_TO_LIBDISPATCH_SOURCE}"
+    -I"${SWIFT_PATH_TO_LIBDISPATCH_BUILD}/src"
+    -I"${SWIFT_PATH_TO_LIBDISPATCH_BUILD}/src/swift"
+    -I"${FOUNDATION_BUILD_DIR}"
+    -I"${FOUNDATION_BUILD_DIR}/usr/lib/swift"
+    -I"${FOUNDATION_BUILD_DIR}/usr/lib/swift/CoreFoundation"
```

Comment:
These can be handled via `target_include_directories`.

Reply:
Thanks! Is there a good way to get the full target name that `add_swift_library` creates?

Reply:
I'm trying to thread through `INCLUDE_DIRECTORIES` in `add_swift_library`, passing it through to `target_include_directories`, and it doesn't actually adjust the command line used to compile `SwiftSyntax`.